### ISSUE TITLE
Adding columns in data table destroys formatting and search

### ISSUE THREAD

**Issue by BRebecca** [almasaeed2010/AdminLTE#1712]
This one has be plaguing me for days. Even using the out of the box, downloaded, unzipped example files pages\tables\data.html -  any time I try to add another column my search vanishes, the ability to sort disappears, and every item in the table is present (no more paging).

This is okay with the appropriate formatting
```

`              <tr>
                  <th>Rendering engine</th>
                  <th>Browser</th>
                  <th>Platform(s)</th>
                  <th>Engine version</th>
                  <th>CSS grade</th>
                </tr>`
```
-------------------------------
This breaks everything 
           

```
               <tr>
                  <th>Rendering engine</th>
                  <th>Browser</th>
                  <th>Platform(s)</th>
                  <th>Engine version</th>
                  <th>CSS grade</th>
                  <th>This won't work</th>
                </tr>

```

I'm really not sure what to do here. I've tried upgrading to the newest database.net and, while some CSS was enhanced, the issue is still the same.

How can I add columns to data tables in AdminLTE while preserving the search and other functionality?

**Comment by vibs2006** [comment]
@BRebecca have you also included this additional column in your **ALL** data rows and Footer as well ?

**Comment by BRebecca** [comment]
I'm both grateful and embarrassed in regards to the resolution.

@vibs2006 I did not have the footer or php listed correctly. I tried creating headers just to see how the formatting would look and did not append the additional columns with php - just left them blank to see how the headers would look.

Everything is in place and working fine. Thanks for your comment and smacking some common sense into me :)
